src/App.tsx

```diff
@@ -17,18 +66,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onChangeQuery={onChangeQuery}
+                resetQuery={resetQuery}
+                setFilterType={setFilterType}
+                filterType={filterType}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
```

Comment:
it isn't correct, why do you show TodoList when isLoading === true?
isLoading === true ? <Loader /> : ...